### ISSUE TITLE
CVE-2022-28391, ALPINE-13661

### ISSUE THREAD

**Issue by achifal** [nginxinc/docker-nginx#647]
`nginx:1.21.6-alpine`
```
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| STATUS     | CVE SEVERITY           | PACKAGE NAME | PACKAGE VERSION | CVE DESCRIPTION                                               |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2022-28391 | busybox      | 1.34.1-r4       | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-28391 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown ALPINE-13661   | busybox      | 1.34.1-r4       | https://cve.mitre.org/cgi-bin/cvename.cgi?name=ALPINE-13661   |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
```

**Comment by adamdecaf** [comment]
If I open a PR would it be merged? I can sign a CLA if needed.

**Comment by yosifkit** [comment]
:confused: What would the PR do since there is no `busybox` package update available?

```console
$ docker run -it --rm nginx:1.21.6-alpine sh
Unable to find image 'nginx:1.21.6-alpine' locally
1.21.6-alpine: Pulling from library/nginx
df9b9388f04a: Already exists 
5867cba5fcbd: Already exists 
4b639e65cb3b: Already exists 
061ed9e2b976: Already exists 
bc19f3e8eeb1: Already exists 
4071be97c256: Already exists 
Digest: sha256:5a0df7fb7c8c03e4158ae9974bfbd6a15da2bdfdeded4fb694367ec812325d31
Status: Downloaded newer image for nginx:1.21.6-alpine
/ # apk --no-cache upgrade
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/community/x86_64/APKINDEX.tar.gz
(1/1) Upgrading xz-libs (5.2.5-r0 -> 5.2.5-r1)
OK: 26 MiB in 42 packages
/ # 
```
(note: for official images, we try to ensure that security updates are addressed in the image in which the package is introduced in order to minimize resulting image size and to ensure that every dependent image also gets the update)

The attack surface seems very narrow and practically non-existent on CVE-2022-28391 for any usage of `nginx` since the user is not likely running an interactive shell and using `netstat` to print a DNS PTR.
>allows remote attackers to execute arbitrary code if netstat is used to print a DNS PTR record's value to a VT compatible terminal. Alternatively, the attacker could choose to change the terminal's colors

---

But, it turns out this is already fixed since [`1.34.1-r5` has the fix](https://gitlab.alpinelinux.org/alpine/aports/-/commit/2745de7e1b09e663b477a8141b84f7d81a049963) and so https://github.com/docker-library/official-images/pull/12184 fixed all alpine-based images (since official-images rebuild when the base image is updated).

```console
$ docker run -it --rm nginx:1.21.6-alpine sh
/ # apk info busybox
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.15/main: No such file or directory
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.15/community: No such file or directory
busybox-1.34.1-r5 description:
Size optimized toolbox of many common UNIX utilities

busybox-1.34.1-r5 webpage:
https://busybox.net/

busybox-1.34.1-r5 installed size:
924 KiB

```

**Comment by thresheek** [comment]
All fixed in both `nginx:alpine` and `nginx:stable-alpine`.  Thanks to Docker Official Library maintainers!
